types/validator_set.go

```diff
@@ -111,6 +94,26 @@ func (vals *ValidatorSet) IncrementProposerPriority(times int) {
 	vals.Proposer = proposer
 }
 
+func (vals *ValidatorSet) RescalePriorities(diffMax int64) {
+	// NOTE: This check is merely a sanity check which could be
+	// removed if all tests would init. voting power appropriately;
+	// i.e. diffMax should always be > 0
+	if diffMax == 0 {
```

Comment:
> // i.e. diffMax should always be > 0

shouldn't this be `if diffMax <= 0 {` then?

Reply:
`diffMax` is (currently) computed as `2 * vals.TotalVotingPower()`. The voting power is expected to be a number > 0. It can only be zero in some test-cases where the voting power isn't properly initialised But as votingPower is an int64 checking for > 0 still makes sense; see https://github.com/tendermint/tendermint/pull/3049#discussion_r248684579.